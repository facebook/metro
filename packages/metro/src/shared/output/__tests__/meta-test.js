/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @format
 * @emails oncall+js_foundation
 * @flow strict
 */

'use strict';

const meta = require('../meta');

it('exports the block list creator', () => {
  expect(meta('some formatted code', 'utf8')).toMatchSnapshot();
});
