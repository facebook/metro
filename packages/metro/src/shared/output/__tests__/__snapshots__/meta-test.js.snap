// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports the block list creator 1`] = `
Object {
  "data": Array [
    56,
    131,
    13,
    89,
    254,
    63,
    108,
    210,
    213,
    112,
    53,
    173,
    11,
    21,
    31,
    184,
    133,
    221,
    114,
    86,
    1,
  ],
  "type": "Buffer",
}
`;
